car_manager.py

```diff
@@ -0,0 +1,38 @@
+import random
+from turtle import Turtle
+COLORS = ["red", "orange", "yellow", "green", "blue", "purple"]
+STARTING_MOVE_DISTANCE = 5
+MOVE_INCREMENT = 10
+
+
+class CarManager(Turtle):
```

Comment:
Also good for organizing the manager into its own class. It'll be easier to use in other places